### ISSUE TITLE
Suggestion for potential improvement for `PHPMailer::getTranslations()`

### ISSUE THREAD

**Issue by jrfnl** [PHPMailer/PHPMailer#2419]
Encountered while writing tests for the logic involved in localizing the error messages.

**_Note: I'd be happy to make the changes I'm suggesting here. Just wanted to allow for discussion before sending in a PR._**

The `public` `PHPMailer::getTranslations()` method is defined as such:
https://github.com/PHPMailer/PHPMailer/blob/6dc419aeadf21d99b8c12e8e139383c394355784/src/PHPMailer.php#L2281-L2289

... with the `$language` property defaulting to an empty array.
https://github.com/PHPMailer/PHPMailer/blob/6dc419aeadf21d99b8c12e8e139383c394355784/src/PHPMailer.php#L696

This means that when no call has been made to `PHPMailer::setLanguage()`, the `PHPMailer::getTranslations()` method will return an empty array.

This feels inconsistent with the handling of this in the (`protected`) `PHPMailer::lang()` method, which automatically sets the language strings to the default language if the translations array is empty:
https://github.com/PHPMailer/PHPMailer/blob/6dc419aeadf21d99b8c12e8e139383c394355784/src/PHPMailer.php#L4046-L4050

I'd like to suggest that the `PHPMailer::getTranslations()` method should always return an array with the text strings and if no call has been made to `PHPMailer::setLanguage()` before the call to `PHPMailer::getTranslations()`, this array will contain the default English language strings.

So basically, I'm proposing to change the `PHPMailer::getTranslations()` method to this:
```php
    public function getTranslations()
    {
        if (empty($this->language)) {
            $this->setLanguage(); // Set the default language.
        }

        return $this->language;
    }
```

Opinions ?

**Comment by Synchro** [comment]
Indeed I don't see any advantage in returning that empty array, so your suggestion seems very sensible.
